src/coreclr/vm/jitinterface.cpp

```diff
@@ -7142,6 +7142,27 @@ bool getILIntrinsicImplementationForInterlocked(MethodDesc * ftn,
     return true;
 }
 
+bool IsBitwiseEquatable(TypeHandle typeHandle, MethodTable * methodTable)
+{
+    if (!methodTable->IsValueType() || methodTable->ContainsPointers())
```

Comment:
I don't see a reason to block this for types with unmanaged pointers?

Reply:
This change does not block it for unmanaged pointers.

`MethodTable::ContainsPointers` returns true for managed object references only.